scripts/shopify/name-remaining.sh

```
@@ -0,0 +1,46 @@
+#   • stopping it at any point loses nothing
+#   • starting it again resumes exactly where it stopped
+#   • nothing is ever named or charged for twice
+#
+# Safe to run under nohup, in a screen, or from cron. Ctrl-C is safe.
+#
+#   bash scripts/shopify/name-remaining.sh
+#
+# Requires GEMINI_API_KEY in the repo-root .env (or the environment), and
+# Application Default Credentials for RTDB.
+set -u
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)